manifests/forward.pp

```diff
@@ -5,6 +5,7 @@
 define unbound::forward (
   $address,
   $zone = $name,
+  $forward_first = 'no'
```

Comment:
Want me to validate_array on $address while I'm in there? Looks like the template will only use it if it's an array.